File: lab-2/src/pokemons/Gabite.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Gabite extends Gible {
    public Gabite (String name, int level) {
        super(name, level);
        setStats(68,90,65,50,55,82);
        setType(Type.DRAGON, Type.GROUND);
        setMove(new HealPulse());
    }
}

File: lab-2-kotlin/src/pokemons/Gabite.kt
package pokemons

import moves.HealPulse

import ru.ifmo.se.pokemon.Type

open class Gabite(name: String?, level: Int) : Gible(name, level) {
    init {
        setStats(68.0, 90.0, 65.0, 50.0, 55.0, 82.0)
        setType(Type.DRAGON, Type.GROUND)
        setMove(HealPulse())
    }
}